File: 04-2-One2NineAppWithRoom_kt2/app/src/main/java/com/example/one2nineappwithroom_kt2/repository/Score.kt
package com.example.one2nineappwithroom_kt2.repository

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
data class Score(
    @PrimaryKey(autoGenerate = true)
    val id: Int = 0,
    val playerName: String,
    val time: Float,
    val whenPlayed: String
)

File: flutter_projects/04-1-one2nineapp_fl/lib/persistence/score.dart
import 'package:floor/floor.dart';

@entity
class Score {
  @PrimaryKey(autoGenerate: true)
  int? id;

  final String playerName;

  final double time;

  final String whenPlayed;

  Score(this.playerName, this.time, this.whenPlayed);
}